README.md

```diff
@@ -1,7 +1,7 @@
 <br /><br />
 <p align="center">
-    <img src="docs/images/header_light.svg#gh-light-mode-only" width="70%"/>
-    <img src="docs/images/header_dark.svg#gh-dark-mode-only" width="70%"/>
+    <img src="https://docs.mosaicml.com/en/stable/_static/header_light.svg#gh-light-mode-only" class="only-light" width="70%"/>
```

Comment:
Need to use absolute URLs so the logo will show on our pypi page. See https://pypi.org/project/mosaicml/

Reply:
Note: going to serve images from the google bucket in the interim, until the branch is merged into main, and then do a seperate PR later to serve from the docs site. This will ensure no downtime for the image display.

Reply:
See #759